frontend/src/app/facebook/facebook.component.ts

```diff
@@ -15,14 +15,23 @@ import {ApiService} from "../api.service";
 export class FacebookComponent implements OnInit {
 
     $users: any;
+    promotions = [];
     p = 0; // The current page
 
 
-    constructor(private apiService: ApiService) {
+    constructor(private api: ApiService) {
     }
 
     ngOnInit() {
-        this.$users = this.apiService.getUsers();
+        this.$users = this.api.getUsers();
```

Comment:
I think we should keep the "Service" and "Components" suffixes, it is easier to understand